### PR TITLE
add transaction streaming support

### PR DESCRIPTION
**Pull request by fardream**
Hi,

I have added the support for transaction streaming. It can be turned on by setting a topic for transaction.

### CODE REVIEW

File: proto/event.proto [hunk 1]
```diff
@@ -52,3 +52,117 @@ enum SlotStatus {
...
+  bool is_simple_vote_transaction = 3;
+  repeated bytes signatures = 4;
+}
+
+message InnerInstruction {
+  uint32 index = 1;
+  repeated CompiledInstruction instructions = 2;
+}
+
+// F64 is like google.protobuf.DoubleValue, but avoid importing another proto.
```
Comment (terorie) [inline proto/event.proto:120]: Importing `google.protobuf.DoubleValue` is fine. Looks like it is bundled with `prost`: https://github.com/tokio-rs/prost/blob/fc9fbd9823f93774004975f7794dac112a2ff4ef/prost-build/third-party/include/google/protobuf/wrappers.proto

File: Cargo.toml [hunk 2]
```diff
@@ -16,6 +16,7 @@ rdkafka = { version = "0.28", features = ["ssl-vendored", "sasl"] }
 solana-geyser-plugin-interface = { version = "=1.10.9" }
 solana-logger = { version = "=1.10.9" }
 solana-program = { version = "=1.10.9" }
+solana-transaction-status = { version = "=1.10.9" }
```
Comment (terorie) [inline Cargo.toml:19]: These deps have been updated in the v1.10 branch. Would you mind rebasing?

File: proto/event.proto (hunk 1 above)
Comment (fardream) [reply proto/event.proto]: > Importing `google.protobuf.DoubleValue` is fine. Looks like it is bundled with `prost`: https://github.com/tokio-rs/prost/blob/fc9fbd9823f93774004975f7794dac112a2ff4ef/prost-build/third-party/include/google/protobuf/wrappers.proto

Thanks. Updated with `google.protobuf.DoubleValue`

**Comment by terorie** [comment]
Thanks a lot for the contribution! I'm testing this on a mainnet validator today and will be merging if everything works fine.

**Pull request merged**